account_multicompany_code/models/__init__.py

```diff
@@ -1,2 +1,3 @@
 from . import res_company
 from . import account
+from . import journal
```

Comment:
```suggestion
from . import account_journal
```

And rename account to its corresponding model.

Nitpick: also sort alphabetically